src/libos/crates/async-rt/src/lib.rs

```diff
@@ -15,10 +15,9 @@
 #![feature(thread_local)]
 #![feature(const_fn_fn_ptr_basics)]
 #![feature(const_fn_trait_bound)]
-#![feature(duration_zero)]
 #![feature(duration_constants)]
-#![feature(duration_saturating_ops)]
 #![feature(get_mut_unchecked)]
+#![allow(dead_code)]
```

Comment:
What is the dead code? Leave dead code here seems not a good thing

Reply:
Because there are two schedulers in this crate but only one will be used.
Without this, many "unused" warnings will be output.